init.lua

```diff
@@ -17,9 +17,18 @@
 ]]
 
 -- Thank you, ApolloX, for adding this.
+-- ApollloX: No problem, glade to help.
 
 local player_lights = {}
-local PLAYER_EYE_POS = 1.5 -- Move this possibly to code generation (rather than ugly hardcode)
+
+local function get_eye_pos(player)
+	if not player or not minetest.is_player(player) then
+		return -- Invalid player object
+	end
+	local props = player:get_properties()
```

Comment:
I remember I did something like this before, just took awhile remembering which of my mods I made use of it in. (rebreather if your wondering where I did this too)